src/Parquet.Runner/Parquet.Runner.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>net45</TargetFramework>
```

Comment:
FYI, moving away from .Net Core will definitely be an issue, but at least your unblocked for now.

Reply:
reverted to netcoreapp2.0